### PR TITLE
remove arrows

### PR DESCRIPTION
**Pull request by kentis**

### CODE REVIEW

File: src/components/Grid/Grid.tsx [hunk 1]
```diff
@@ -203,57 +188,9 @@ export const Grid: FC<GridProps> = ({
 
   const createArrow = useCallback(
     (elementId: string, pointerPosition: Position) => {
-      const isCreatingNewArrow = activeTool === ToolbarButtonType.CreateArrow;
       const isCreatingClassicArrow =
```
Comment (boyum) [inline src/components/Grid/Grid.tsx:191]: Should we rename this to `isCreatingArrow`?

File: src/components/Grid/Grid.tsx [hunk 2]
```diff
@@ -346,17 +282,13 @@ export const Grid: FC<GridProps> = ({
         }
 
         setClassicArrowStartId(null);
```
Comment (boyum) [inline src/components/Grid/Grid.tsx:284]: Can we remove "classic" from all variable/function names?

File: src/components/Grid/Grid.tsx [hunk 3]
```diff
@@ -837,21 +759,11 @@ export const Grid: FC<GridProps> = ({
...
-      if (updatedItem) {
-        console.info("Set OLD Arrow Type", type, id, updatedItem);
-        const newItems = updateArrowType(arrowItems, updatedItem, type, items);
-        console.info("Updated OLD Arrow Type", type, id, newItems);
-        updateArrowItems(newItems);
-        setArrowItems(newItems);
-      }
-
       if (updatedClassicItem) {
         console.info("Set Classic Arrow Type", type, id, updatedClassicItem);
```
Comment (boyum) [inline src/components/Grid/Grid.tsx:768]:
```suggestion
```

**Comment by boyum** [comment]
Also run `npm uninstall react-xarrows`

File: src/components/Grid/Grid.tsx (hunk 1 above)
Comment (kentis) [reply src/components/Grid/Grid.tsx:191]: jepp, egen pr

File: src/components/Grid/Grid.tsx (hunk 2 above)
Comment (kentis) [reply src/components/Grid/Grid.tsx:284]: jepp, i egen pr

File: src/components/MapEditorView/MapEditorView.tsx [hunk 4]
```diff
@@ -177,7 +177,6 @@ export const MapEditorView: React.FC<MapEditorViewProps> = ({
           initialItems={gridItems}
           updateItems={updateItems}
           initialArrowItems={arrowItems}
```
Comment (boyum) [inline src/components/MapEditorView/MapEditorView.tsx:179]: Do we need `initialArrowItems`?

Comment (kentis) [reply src/components/MapEditorView/MapEditorView.tsx:179]: yes, in order to edit arows, we need to know which exist already.

**Review by boyum — APPROVED** [review]

**Pull request merged**